src/components/Common/Footer/Footer.scss

```diff
@@ -0,0 +1,42 @@
+// Footer.scss
+
+@import "../../../themes/mixins";
+
+.footer-container {
+  display: flex;
+  padding: 20px 232px 24px 19px;
+
+  @include desktop {
```

Comment:
The desktop-specific styles are applied directly within the nested selectors. While this is functional, it can make the code harder to read and maintain. Consider using media queries at the root level of the stylesheet to separate desktop-specific styles more cleanly.